site/sfguides/src/data_mapping_in_native_apps/data_mapping_in_native_apps.md

```diff
@@ -60,27 +61,58 @@ The application provides a user interface which allows the consumer of the appli
 
 The first solution is not really what we want to be doing because the consumer will potentially have to create objects on their Snowflake instance just to satisfy the application's requirements, so this Quickstart will deliver the second solution.
 
+> aside positive
+> 
+> **Note** - The following steps explain how to create the app using the Snowflake CLI, as this option presents as a faster, more straightforward way to build Native Apps. You can also accomplish it by manually creating the folder structure and executing the SQL commands in Snowsight, but this alternative is out of the scope for this Quickstart.
+
+The files explained in the following steps, are already present in the project folder, and it is merely a deeper explanation of its contents.
+
+
+<!-- ------------------------ -->
+## Snowflake CLI First Steps
```

Comment:
I suggest we add a short paragraph that links to the official Snowflake CLI docs as to how to set up connections as a prerequisite.